x-pack/plugins/observability/public/components/shared/core_web_vitals/web_core_vitals_title.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiButtonIcon, EuiLink, EuiPopover, EuiText, EuiTitle } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+const CORE_WEB_VITALS = i18n.translate('xpack.observability.ux.coreWebVitals', {
+  defaultMessage: 'Core web vitals',
+});
+
+export function WebCoreVitalsTitle() {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false);
+
+  const closePopover = () => setIsPopoverOpen(false);
+
+  return (
+    <EuiTitle size="xs">
+      <h3>
+        {CORE_WEB_VITALS}
+        <EuiPopover
+          isOpen={isPopoverOpen}
+          button={
+            <EuiButtonIcon
+              onClick={() => setIsPopoverOpen(true)}
+              color={'text'}
+              iconType={'questionInCircle'}
+            />
+          }
+          closePopover={closePopover}
+        >
+          <div style={{ width: '300px' }}>
```

Comment:
I don't think it is needed, is it?

Reply:
i think i must have added it for a reason, it was going into second line i think with default width.

Reply:
I removed it and it looks like it keeps working. 
<img width="333" alt="Screenshot 2020-10-05 at 11 44 22" src="https://user-images.githubusercontent.com/55978943/95064528-28277100-0700-11eb-80a2-183b562618ee.png">

Reply:
ahh ok my bad, i will remove it.